### PR TITLE
Swagger updates to introduce cosmos-db api version 2020-03-01, and add PublicNetworkAccess property

### PR DESCRIPTION
**Pull request by ayanMSFT**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by Liphi** [comment]
the api ver is Mar 1 03-01, not Jan 3 01-03. Can u correct it?

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/cosmos-db.json [hunk 1]
```diff
@@ -0,0 +1,7248 @@
...
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
```
Comment (lirenhe) [inline specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/cosmos-db.json]: Could you add the default response?

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/cosmos-db.json [hunk 2]
```diff
@@ -0,0 +1,7248 @@
...
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DatabaseAccountUpdateParameters"
+            },
+            "description": "The parameters to provide for the current database account."
+          }
+        ],
+        "responses": {
+          "200": {
```
Comment (lirenhe) [inline specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/cosmos-db.json]: For long running operation, will you also add the response code of 201, 202?

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/cosmos-db.json [hunk 3]
```diff
@@ -0,0 +1,7248 @@
...
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The account name is valid but is already in use."
+          },
+          "404": {
```
Comment (lirenhe) [inline specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/cosmos-db.json]: Is 404  the right error code in this case?

**Review by ryansbenson — COMMENTED** [review]
Approved from ARMs side

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/cosmos-db.json (hunk 3 above)
Comment (ayanMSFT) [reply specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/cosmos-db.json]: That is what the API returns currently for all other versions. Its not changing in this version update.

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/cosmos-db.json (hunk 2 above)
Comment (ayanMSFT) [reply specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/cosmos-db.json]: The previous versions of this API return 200. It hasnt changed with this version update.

Comment (lirenhe) [reply specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/cosmos-db.json]: I am trying to understand the behavior of your service. So for this API, will your service only return the response code of 200 only?

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/cosmos-db.json (hunk 1 above)
Comment (lirenhe) [reply specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/cosmos-db.json]: I saw you added and then removed the default response, what is the major reason for that

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/cosmos-db.json (hunk 2 above)
Comment (ayanMSFT) [reply specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/cosmos-db.json]: Yes, it does only return 200

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/cosmos-db.json (hunk 1 above)
Comment (ayanMSFT) [reply specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/cosmos-db.json]: I checked the implementation. The only other response code we could return is 404, I added it.

**Review by lirenhe — APPROVED** [review]

**Pull request merged**